packages/eas-cli/src/metadata/upload.ts

```diff
@@ -22,18 +23,29 @@ export async function uploadMetadataAsync(
     throw new MetadataValidationError(`Store configuration file not found "${filePath}"`);
   }
 
+  const fileData = await fs.readJson(filePath);
+  const { valid, errors: validationErrors } = validateConfig(fileData);
+  if (!valid) {
+    const error = new MetadataValidationError(`Store configuration errors found`, validationErrors);
+    logMetadataValidationError(error);
+    Log.newLine();
+    Log.warn(
+      'The current store configuration might fail during syncing or could fail review when submitted without additional changes.'
```

Comment:
I'm not an expert on english but this sentence sounds weird to me.

Sth like `... or can result in rejected submission`?

Reply:
```suggestion
      'Without further updates, the current store configuration may fail to be synchronized with the App Store or pass App Store review.'
```

Something like this could work.